generate-manifest.py

```diff
@@ -107,6 +117,22 @@ def edit_manifest(data, arch, branch, runtime_version):
             data['modules'].insert(0, gtk_module)
             break
 
+    # GNOME SDK's WebkitGTK+ module is only needed for our arm SDK
+    if arch != 'arm':
+        return
+    version = runtime_version.replace('.', '-')
+    u = request.urlopen(GNOME_MANIFEST_URL.format(version=version))
+    sdk_manifest = json.loads(re.sub(r'(^|\s)/\*.*?\*/', '', u.read().decode('utf-8'), flags=re.DOTALL))
+    for m in sdk_manifest['modules']:
+        if m['name'] == 'WebKitGTK+':
+            webkitgtk_module = m
+            for arch in webkitgtk_module['build-options']['arch']:
+                if arch not in webkitgtk_config_opts:
+                    continue
+                webkitgtk_module['build-options']['arch'][arch]['config-opts'] += webkitgtk_config_opts[arch]
+            data['modules'].insert(0, webkitgtk_module)
```

Comment:
Don't we want to build WebKit Gtk+ after our Gtk+? This would appear to place it first in the generated module list. Maybe insert(1)?

Reply:
good point.